pkg/signatory/signatory.go

```diff
@@ -154,9 +154,9 @@ func (s *Signatory) matchFilter(msg tezos.UnsignedMessage, policy *config.TezosP
 	if ops, ok := msg.(*tezos.UnsignedOperation); ok {
 		for _, op := range ops.Contents {
 			opKind := op.OperationKind()
-            if (kind != "generic" && opKind != "") {
-                return fmt.Errorf("request kind `%s' can not have operations", kind)
-            }
+			if kind != "generic" && opKind != "" {
+				return fmt.Errorf("request kind `%s' can not have operations", kind)
```

Comment:
This change is reverting to the old behaviour. You want to remove this particular change.